src/lando/main/management/commands/start_uplift_worker.py

```diff
@@ -13,3 +16,26 @@
 class Command(StartWorkerCommand):
     help = "Start an uplift worker."
     worker_class = UpliftWorker
+
+    def setup_moz_phab_config(self):
+        """Copy the moz-phab config file to the user's home directory."""
+        source_config = settings.BASE_DIR / "api/legacy/workers/config/moz-phab-config"
```

Comment:
I think it would be better for the config to actually be generated and written to disk here, rather than being checked in as a data file that's copied over. It's just a little weird because of the way Python treats non-python files in packages, I don't think we should include it in this way here. Alternatively we could include it as a data file (i.e., in pyproject.toml) but that's probably a little more work than generating it in this module.

Reply:
Moved the config definition to a string in the `start_uplift_worker.py` command and then write the text out into the destination on worker startup. :+1: